src/main/java/frc/robot/commands/MakeIntakeMotorSpin.java

```diff
@@ -45,7 +45,7 @@ public void initialize() {
     @Override
     public void execute() {
         if ( intakeSubsystem != null ) {
-            intakeSubsystem.setMotorRPM(targetSpeed);
+            intakeSubsystem.setVoltage(targetSpeed);
```

Comment:
update targetSpeed to targetVoltage to avoid confusion

Reply:
Done